packages/slice-machine/lib/env/manifest.ts

```diff
@@ -49,63 +55,6 @@ export function extractRepo(parsedRepo: ParseResult): string | undefined {
   }
 }
 
-function validateEndpoint(endpoint: string, parsedRepo: ParseResult): boolean {
```

Comment:
nice to see so much removal.
And on top of that since it's in the validator we won't have to worry about doing that elsewhere or on another app.